src/systems/light.js

```diff
@@ -73,5 +98,17 @@ module.exports.System = registerSystem('light', {
     sceneEl.appendChild(directionalLight);
 
     this.defaultLights = true;
+  },
+
+  /**
+   * Enables/disables the renderer shadow map.
+   * @param {boolean} enabled
+   */
+  setShadowMapEnabled: function (enabled) {
+    var renderer = this.sceneEl.renderer;
+    this.shadowMapEnabled = enabled;
+    if (renderer) {
+      renderer.shadowMap.enabled = enabled;
+    }
```

Comment:
You need to update materials in a scene to dynamically handle shadow on/off 
after scene is rendered once if you switch `shadowMap.enabled`.

Reference:
Run the following commands one by one on the console at
https://threejs.org/examples/webgl_animation_skinning_morph.html

    renderer.shadowMap.enabled = false;
    mesh.material.materials[0].needsUpdate = true;
    renderer.shadowMap.enabled = true;
    mesh.material.materials[0].needsUpdate = true;